apps/lockfile-explorer/src/start.ts

```diff
@@ -103,7 +105,30 @@ function startApp(debugMode: boolean): void {
 
   app.get('/api/lockfile', async (req: express.Request, res: express.Response) => {
     const pnpmLockfileText: string = await FileSystem.readFileAsync(appState.pnpmLockfileLocation);
-    const doc = yaml.load(pnpmLockfileText);
+    const doc = yaml.load(pnpmLockfileText) as Lockfile;
+    const { packages, lockfileVersion } = doc;
+
+    let shrinkwrapFileMajorVersion: number;
+    if (typeof lockfileVersion === 'string') {
+      shrinkwrapFileMajorVersion = parseInt(lockfileVersion.substring(0, lockfileVersion.indexOf('.')), 10);
+    } else if (typeof lockfileVersion === 'number') {
+      shrinkwrapFileMajorVersion = lockfileVersion;
```

Comment:
If it is encoded as a `number`, then `shrinkwrapFileMajorVersion` should probably be `Math.floor(lockfileVersion)` right?